rpcs3/Emu/RSX/rsx_decode.h

```diff
@@ -4064,21 +4078,23 @@ struct registers_decoder<NV3089_IMAGE_IN>
 	public:
 		decoded_type(u32 raw_value) { m_data.raw_value = raw_value; }
 
-		u16 x() const
+		// x and y given as 16 bit fixed point
+
+		f32 x() const
 		{
-			return m_data.x;
+			return m_data.x / 16.f;
```

Comment:
Why divide by 16.0 here?  i can't follow it.

Reply:
its a 16 bit fixed point number, so it needs to be divided to turn back into the correct float, also confirmed in psl1ght sources https://github.com/ps3dev/PSL1GHT/blob/534e58950732c54dc6a553910b653c99ba6e9edc/samples/graphics/blitting/source/main.c#L180

Reply:
Maybe make a function for it? it'd be better if it were explicit